adapter/internal/api/apis_impl.go

```diff
@@ -223,7 +251,7 @@ func validateAndUpdateXds(apiProject mgw.ProjectAPI, override *bool) (err error)
 
 	// TODO: (renuka) optimize to update cache only once when all internal memory maps are updated
 	for vhost, environments := range vhostToEnvsMap {
-		_, err = xds.UpdateAPI(vhost, apiProject, environments, common.XdsOptions{})
+		_, err = xds.UpdateAPI(vhost, apiProject, environments, common.XdsOptions{}, apiProject.IsPaidOrg)
```

Comment:
```suggestion
		_, err = xds.UpdateAPI(vhost, apiProject, environments, common.XdsOptions{})
```

We are already passing `apiProject` and we can read IsPaidOrg  from `apiProject. IsPaidOrg `